webapp/tests/karma/ts/services/changes.service.spec.ts

```diff
@@ -59,7 +59,10 @@ describe('Changes service', () => {
     store = TestBed.inject(MockStore);
   }));
 
-  afterEach(() => sinon.restore());
+  afterEach(() => {
+    store.resetSelectors();
```

Comment:
This almost seems like the default. What happens if we don't reset selectors?

Reply:
Other spec files will get contaminated stores and have side effects. It won't reset the store before every test like before

Reply:
The good side is that we can now make separate test cases using the store as it changes over actions.

Reply:
So sad. I think we should maybe consider adding a global store property and adding a global afterEach to reset it, at some point.

Reply:
That's a good idea and we can include the `sinon.restore()` as it's very much in every spec file. We can do this in another PR.